.github/pull_request_template.md

```diff
@@ -16,3 +16,4 @@ For all contributors:
 - [ ] You have added tests (when appropriate).
 - [ ] You have added an entry in the CHANGELOG (when appropriate).
 - [ ] You have updated the README or other documentation to account for these changes (when appropriate).
+- [ ] You have either (a) made changes to sprocket.bio [if you are a Sprocket team member] or (b) have suggested any changes you _think_ need to be made to sprocket.bio in the description of your PR [if you are an external contributor].
```

Comment:
```suggestion
- [ ] You have either (a) made a PR for sprocket.bio [if you are a Sprocket team member] or (b) have suggested any changes you _think_ need to be made to sprocket.bio in the description of your PR [if you are an external contributor].
```
As you had it written, it implied to me the sprocket.bio changes need to land before the sprocket PR which seems backwards to me